src/common.c

```diff
@@ -243,7 +243,7 @@ struct GameSamples *readsamples(const char **samplenames,const char *basename)
 	int skipfirst = 0;
 
 	/* if the user doesn't want to use samples, bail */
-	if (!options.use_samples) return 0;
+	/* if (!options.use_samples) return 0; */

```

Comment:
? should be 1 so not sure why this needs commenting out?

Reply:
I can un comment it if you want it is set to one. It was the paths that was the problem if its a game changer for you guts ill remove it. Im pretty sure the option for that went back from the settings menu a while ago @dankcushions  if you want it commented let me know

Reply:
this comes back to my other comment i guess - i don't think we should remove MAME code just because we don't use it - it just seems like busywork to me, and adds clutter to these PRs that are supposed to be solving a stated issue.

but yeah, if you're going to remove it - just remove it. don't comment stuff out. that makes it look like it was previously causing a problem, but there's no explanation as to why. i vote to just leave the line in and do refactors in seperate PRs.